### ISSUE TITLE
eth price in cli should not be displayed for private chains

### ISSUE THREAD

**Issue by debris** [paritytech/parity#6228]
```
2017-08-04 00:19:29   INFO miner  Updated conversion rate to Ξ1 = US$223.41 (0 wei/gas)
```

**Comment by tomusdrw** [comment]
Actually it's valid only for `foundation` chain, should probably use some predefined min gas price for all other chains.

**Comment by keorn** [comment]
Maybe solve it by `Engine` specific default configs? A few other issues exist since the defaults are suited to Ethash.

**Comment by 5chdn** [comment]
+1 for more chain abstraction in the client via config, this could be solved too: #4123
